### ISSUE TITLE
archiving this repo?

### ISSUE THREAD

**Issue by lorenzwalthert** [r-spark/sparklyr.sedona#63]
If I understand correctly, the sparklyr extension is developed in the upstream repo?

**Comment by yitao-li** [comment]
Yes this repo should be archived after I get to publish sparklyr.sedona to CRAN.
